### ISSUE TITLE
Jincor Bug Bounty Program

### ISSUE THREAD

**Issue by pabloruiz55** [JGcarv/ethereum-bounty-hunters#13]
### Description ###

[Reddit post](https://www.reddit.com/r/ethdev/comments/7758bo/jincor_smart_contracts_code_review_bounty_program/)

[Announcement post here](https://bitcointalk.org/index.php?topic=2206571.0)
(Item 11, by the end of the post)

### Scope ###

[Contracts in this folder](https://github.com/JincorTech/ico/tree/develop/contracts)

### Rewards ###

> We invite all developers to review our project's smart contracts and receive reward from 100 and up to 10.000 JCR (1 JCR will cost 1$ on 15 November ICO) depends of how critical is problem which you'll find.

### Timeline ###

No information

Contact

> Our CTO will manually review each feedback in Github. After you released an issue please contact us in Telegram.

**Comment by AndrewBezold** [comment]
No details about a bug bounty anywhere.  On top of that, there's no details about the bug bounty in this post either.  Very low-effort.  If you look at the rest of the posts, you'll see what you should be designing this one after.

**Comment by pabloruiz55** [comment]
@AndrewBezold you are right. I added it as not to lose it and then forgot to complete the description. Just updated it with what they posted.
